### PR TITLE
Add wannierberri

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/lazy_property/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{% set name = "lazy_property" %}
```
Comment (scopatz) [inline recipes/lazy_property/meta.yaml:1]: Can this be `lazy-property` so the name is the same as on pypi?

File: recipes/wannierberri/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  host:
+    - python
+    - pip
+    - numpy
+    - scipy
+    - lazy_property
+    - pyfftw
+    - colorama
+    - termcolor
+    - pyfiglet
```
Comment (scopatz) [inline recipes/wannierberri/meta.yaml:27]: Are all of these needed at package build time?

File: recipes/lazy_property/meta.yaml (hunk 1 above)
Comment (jan-janssen) [reply recipes/lazy_property/meta.yaml]: Done.

File: recipes/wannierberri/meta.yaml (hunk 2 above)
Comment (jan-janssen) [reply recipes/wannierberri/meta.yaml:27]: The issue is they import the package itself during setup https://github.com/stepan-tsirkin/wannier-berri/blob/master/setup.py#L9 so I basically need all dependencies already during the installation.

**Comment by jan-janssen** [comment]
@scopatz Can you take another look at this PR?

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**